sl.c

```diff
@@ -87,6 +91,14 @@ int main(int argc, char *argv[])
             option(argv[i] + 1);
         }
     }
+    if (RAND == 1)
+    {
+        srand(time(NULL));
+        ACCIDENT  = rand() % 2;
```

Comment:
Seems like `|=` might fit better here than simply `=` so we're not overriding other command line flags?